addons/pos_restaurant/tests/test_frontend.py

```diff
@@ -10,12 +10,34 @@ def setUp(self):
         self.env = self.env(user=self.env.ref('base.user_admin'))
         account_obj = self.env['account.account']
 
+        account_receivable = account_obj.create({'code': 'X1012',
+                                                 'name': 'Account Receivable - Test',
+                                                 'user_type_id': self.env.ref(
+                                                     'account.data_account_type_receivable').id,
+                                                 'reconcile': True})
+
         printer = self.env['restaurant.printer'].create({
             'name': 'Kitchen Printer',
             'proxy_ip': 'localhost',
         })
         drinks_category = self.env['pos.category'].create({'name': 'Drinks'})
 
+        main_company = self.env.ref('base.main_company')
+
+        second_cash_journal = self.env['account.journal'].create({
+            'name': 'Cash 2',
+            'code': 'CSH2',
+            'type': 'cash',
+            'company_id': main_company.id
+            })
+
+        second_cash_payment = self.env['pos.payment.method'].create({
```

Comment:
No need to store it into a variable. Style CI will not complain anymore ;)

Reply:
Okay, I removed it :)